src/Sulu/Bundle/MediaBundle/Resources/public/js/components/collections/masonry-decorator/masonry-view.js

```diff
@@ -293,7 +293,7 @@ define([
                     item.id,
                     image,
                     title,
-                    item.fallbackLocale,
+                    item.locale !== this.options.locale ? item.locale : null,
```

Comment:
Is it possible that this change is responsible for the blue badge with the fallback disappearing in the media assignment?
![image](https://cloud.githubusercontent.com/assets/405874/15017694/e59f2c7a-1216-11e6-9c1b-08834a5cd729.png)